Add view-transition-class property (#37901)
mdn/content: files/en-us/web/api/view_transition_api/index.md

- {{cssxref("view-transition-name")}}
  - : Provides the selected element with a separate identifying name and causes it to participate in a separate view transition from the root view transition — or no view transition if the `none` value is specified.
- {{cssxref("view-transition-class")}}

>   - : Provides the selected elements that already have a `view-transition-name` with an additional method of styling.

Reviewer: ```suggestion
  - : Provides an additional method of styling to selected elements that have a `view-transition-name`.
```
